src/components/mobile-menu-container/index.tsx

```diff
@@ -18,6 +23,43 @@ const MOBILE_MENU_MOTION = {
 	},
 }
 
+/**
+ * Handles rendering the Sign In and Sign Up UI elements in mobile viewports.
+ * Intended to be used alongside `MobileMenuContainer`.
+ *
+ * @TODO Render user dropdown disclosure for authenticated users
+ * ref: https://app.asana.com/0/1202097197789424/1202665629707469/f
+ */
+const MobileAuthenticationControls = () => {
+	const { isAuthEnabled, isAuthenticated, isLoading, signIn } =
+		useAuthentication()
+	const shouldShowAuthButtons = isAuthEnabled && !isLoading && !isAuthenticated
```

Comment:
💭 I wonder if it would make sense to add `showAuthUI` to the return value from the `useAuthentication` hook. Not necessarily for this PR but I imagine this is something we'll be defining in a lot of components.

Reply:
Ooooh I really like this idea! 🙌🏻

I think we'd need two variables though:

- `showUnauthenticatedUI`
- `showAuthenticatedUI`

Both of these could be derived by `useAuthentication`, and that would mean greatly reducing references to `isAuthEnabled` which is inevitably going away when we release auth features! ✨ I'll explore adding these in one of my PRs where I need to render something different for authenticated users (like the user dropdown disclosure).